### ISSUE TITLE
[FEATURE] Import from package manager

### ISSUE THREAD

**Issue by jayvdb** [tversteeg/emplace#166]
A larger alternative to solve the same problem as https://github.com/tversteeg/emplace/issues/165 is to import the existing list of packages from the package manager. And an `--all` or multi-select would be very helpful.

**Comment by jayvdb** [comment]
Another use case is when there are installers which wrap package managers.
e.g. I used https://github.com/MenkeTechnologies/zpwr to fast-track installing a bunch of brew dependencies which I will want, but emplace has no visibility of them.

**Comment by tversteeg** [comment]
> A larger alternative to solve the same problem as #165 is to import the existing list of packages from the package manager. And an --all or multi-select would be very helpful.

I agree with the validity of this feature, but it's quite a big task to retroactively add it for all the supported package managers.

> Another use case is when there are installers which wrap package managers.
e.g. I used MenkeTechnologies/zpwr to fast-track installing a bunch of brew dependencies which I will want, but emplace has no visibility of them.

Zwpr can also be added if there's a command to install stuff with that. If there's not and it's just a single configuration file then that makes more sense.

**Comment by jayvdb** [comment]
zwpr is a custom installer script.

**Comment by tversteeg** [comment]
@all-contributors please add @jayvdb for idea

**Comment by jayvdb** [comment]
I get thanked for the idea, and then the idea is marked wontfixed.  Bots are great, arnt they.

**Comment by tversteeg** [comment]
I agree, I've removed the bot.
